style.qss

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright (c) 2016 Shotgun Software Inc.
+ 
+CONFIDENTIAL AND PROPRIETARY
+ 
+This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+Source Code License included in this distribution package. See LICENSE.
+By accessing, using, copying or modifying this work you indicate your 
+agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+not expressly granted therein are reserved by Shotgun Software Inc.
+*/
+
+QLabel {
+    font-family: "Open Sans";
```

Comment:
I've removed it. We didn't need it. I also removed the background-color styling, as that was also not helping us any.